aviary/mission/flops_based/ode/mission_ODE.py

```diff
@@ -152,3 +196,12 @@ def setup(self):
                                ('mass', Dynamic.Mission.MASS)
                            ],
                            promotes_outputs=['initial_mass_residual'])
+
+        self.nonlinear_solver = om.NewtonSolver(solve_subsystems=True,
```

Comment:
Adding the solver has some ramifications. I wondered if you could confine it to a subsystem, but looks like that subsystem would need to contain at least the engine, and that's in the external subsystem group, so there probably isn't anything else we can do.